android/src/main/java/com/unflowreactnativesdk/UnflowModule.kt

```diff
@@ -30,15 +31,20 @@ class UnflowModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
 
     @ReactMethod
     fun setAttributes(attributes: ReadableMap) {
-      val mappedAttributes = attributes as Map<String, String>
+      val mappedAttributes = attributes.toHashMap() as Map<String, String>;
       UnflowSdk.client().setAttributes(attributes = mappedAttributes)
     }
 
     @ReactMethod
     fun setCustomFonts(fonts: ReadableMap) {
-//      TODO("Pull font family from assets")
-//      UnflowSdk.client().setCustomFonts(fonts = Fonts(
-//
-//      ))
+      val context = reactApplicationContext
+      UnflowSdk.client().setCustomFonts(fonts = Fonts(
+          title = if (fonts.hasKey("title")) context.resources.getIdentifier(fonts.getString("title"), "font", context.packageName) else null,
```

Comment:
this relies on the fonts being in the `res` folder. what if they're in `assets` instead?

i think if we have a way to pass them through as resolved types it might be less brittle

Reply:
I think we can give good documentation around the fact that the fonts need to live in the `res` folder, especially if that is the convention on Android. Would rather optimize for the native SDK staying as is until we see someone use this

Reply:
cool. my only concern with that is that react native users may have a different convention (assets folder for example), but happy to roll with it for now.

maybe just extract that line into an extension function:
```
fun ReadableMap.getFontResId(context: Context, key: String): Int? {
  if (!hasKey("title")) return null
  return context.resources.getIdentifier(getString(fontName), "font", context.packageName)
}
```